_includes/help-modal.html

```diff
@@ -89,7 +89,7 @@
           <!-- Tab content -->
           <div id="MesheryUI" class="tabcontent">
             <h3>Meshery UI</h3>
-            <p>To import designs, visit <a href="https://docs.meshery.io/guides/meshery-design#import-your-designs">docs</a></p>
+            <p>To import designs, visit <a href="https://docs.meshery.io/guides/configuration-management/importing-apps#import-apps-using-meshery-ui" target="_blank">docs</a></p>
```

Comment:
`target="_blank"`
I'm not sure if we would want these help pages to open in new tab. @sudhanshutech might have a different say here.

Reply:
We can open docs in new tab, there are many benefits of it: 
It allows users to maintain their context on the original page. They can refer back to the documentation while still having the original page accessible. In this way we are not forcing user to navigate away from the current page.